### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by waseembrare**
Please can both html and css be reviewed. The media queries haven't been done so it's just main desktop format so far. Thanks in advance! 

nb. I've botched my pull requests prior, so some of the code miught not allow you to make specific line comments. If this is the case, you can still make general comments I believe 😨

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -18,6 +18,15 @@ font-family: 'MuseoModerno', cursive;
...
+ * Generated by Animista on 2021-1-20 11:9:52
+ * Licensed under FreeBSD License.
+ * See http://animista.net/license for more info. 
+ * w: http://animista.net, t: @cssanimista
+ * ---------------------------------------------- */
+
+@-webkit-keyframes text-focus-in{0%{-webkit-filter:blur(12px);filter:blur(12px);opacity:0}100%{-webkit-filter:blur(0);filter:blur(0);opacity:1}}@keyframes text-focus-in{0%{-webkit-filter:blur(12px);filter:blur(12px);opacity:0}100%{-webkit-filter:blur(0);filter:blur(0);opacity:1}}
+
 *, *::before, *::after { 
 	box-sizing: border-box;
```
Comment (mtmgoliath) [inline css/styles.css:31]: I would include font-size: 10px here and then use rem sizing for the rest of the code. I will comment rem when i see px that should be changed

File: css/styles.css [hunk 2]
```diff
@@ -91,19 +96,25 @@ i:hover {
 	transform: scale(1.2) rotate(5deg);
 }
 
+.text-focus-in {
+	-webkit-animation: text-focus-in 3s cubic-bezier(0.550, 0.085, 0.680, 0.530) both;
+	        animation: text-focus-in 3s cubic-bezier(0.550, 0.085, 0.680, 0.530) both;
+}
+
```
Comment (mtmgoliath) [inline css/styles.css:103]: this looks cool! gotta learn me animations

File: css/styles.css [hunk 3]
```diff
@@ -91,19 +96,25 @@ i:hover {
...
 	font-size: 95px;
   	font-family: 'Stijla', 'Baumans', cursive;
 	line-height: 0;
-/*	margin-top: 40px;
-*/}
+	display: none;
+}
 
 h3 {
 	font-size: 40px;
```
Comment (mtmgoliath) [inline css/styles.css:120]: 4rem

File: css/styles.css [hunk 4]
```diff
@@ -127,10 +138,6 @@ nav {
 	margin-right: 15px;
```
Comment (mtmgoliath) [inline css/styles.css:138]: 1.5rem

File: css/styles.css [hunk 5]
```diff
@@ -127,10 +138,6 @@ nav {
...
 }
 
-/*p>nav:hover {
-	color: #B51515;
-}*/
-
 .section-heading {
 	font-family: 'Quicksand', sans-serif;
 	border-bottom: 2px solid black;
```
Comment (mtmgoliath) [inline css/styles.css:143]: 0.2rem

File: index.html [hunk 6]
```diff
@@ -31,8 +31,13 @@
...
 		<section class="jumbotron">
-			<div><h1>waseembrare</h1></div>
-			<div><h2>//developer</h2></div>
+			<div>
+				<img class="site-heading text-focus-in" src="assets/site-heading.png" alt="Website title: Waseem Brare, developer" width="900">
+				<h1>waseembrare</h1>
+			</div>
+			<div>
+				<h2>//developer</h2>
+			</div>
```
Comment (mtmgoliath) [inline index.html:40]: more semantic/descriptive naming for the cta class would be good. what does it stand for?
using cta-left/right not best practice incase of styling changes

**Review by mtmgoliath — CHANGES_REQUESTED** [review]
looking good with some cool effects.
2 main things:
---change from px to rem for sizing
---better semantic names/ descript for cta class and any others that aren't clear

**Review by mtmgoliath — APPROVED** [review]
Good stuff. Noticed you didnt change the font-size in body to 1.4rem but just deleted the 14px line. this might have knock on effects. worth checking out

File: index.html (hunk 6 above)
Comment (waseembrare) [reply index.html:40]: changed left-cta to cta-sub and right-cta to cta-main for more semantic class

**Pull request merged**